.github/workflows/zowe-cli-plugin.yml

```diff
@@ -97,9 +88,15 @@ jobs:
 
     - name: Install Dependencies
       run: npm ci
+      
+    - name: Update Dependencies
+      id: npm-update
+      uses: zowe-actions/octorelease/script@v1
+      with:
+        script: npmUpdate

```

Comment:
sorry for the confusion, this step should be in the previous job 😢 
ideally right after `npm ci` (where it was) 😋